app/services/account_reset/cancel_request_for_user.rb

```diff
@@ -0,0 +1,20 @@
+module AccountReset
+  class CancelRequestForUser
+    attr_reader :user
+
+    def initialize(user)
+      @user = user
+    end
+
+    def call
+      account_reset_request.update!(cancelled_at: Time.zone.now)
```

Comment:
I would consider making `Time.zone.now` a default parameter to `call` like this so you can inject it directly in tests:

```
def call(now: Time.zone.now)
```